README.md

```diff
@@ -75,7 +75,6 @@ This requires:
 ## Integrations
 
 The following integrations are provided:
-- Auth0, can be disabled with `AUTH0_CLIENT_CREATION_ENABLED`
```

Comment:
I think this can be replaced with Keycloak, it should also have a flag to control the creation of keys. For example when doing something in batch locally or on ACC and you don't want to pollute the external Keycloak database.